draft-ietf-quic-v2.md

```diff
@@ -140,8 +140,17 @@ Compatible version negotiation between versions 1 and 2 follow the same
 requirements in either direction. This section uses the terms "original
 version" and "negotiated version" from {{QUIC-VN}}.
 
-If the server elects to send a Retry packet, it MUST do so using the original
-version.
+If the server sends a Retry packet, it MUST use the original version. The client
+MUST NOT use a different version in the subsequent Initial that contains the
```

Comment:
This MUST NOT doesn't really make much sense as the client won't have any information from the server that might even suggest the possibility that a different version would work.

Reply:
I agree that this is a silly thing for the client to do, but I'm all for forbidding silly things to simplify the code. It does give the freedom for the server to have version-specific retry tokens, which is useless in v2 but probably establishes a better pattern for other versions.